### PR TITLE
Add interfaces for read and write databases

### PR DESCRIPTION
**Pull request by Kemosabert**
Since we can connect to DB's for writing and reading, we'd like to have some type safety for this during the tests.

### CODE REVIEW

**Review by julien-leclercq — COMMENTED** [review]
Do we need to have some changes in Sandbox as well ? (for the connection no ?)

**Comment by Kemosabert** [comment]
Hehe, I tried with the types and it was fine, but tests were getting stuck 😬 Good catch, that would have been annoying 😄

File: platform/database/tests/database_test.go [hunk 1]
```diff
@@ -810,4 +810,140 @@ func TestProdDatabase(t *testing.T) {
...
+
+		if err := runSelectTest(sandBoxReadDB, thirdData); err != nil {
+			t.Fatalf("The select command failed: %v", err)
+		}
+
+		if err := runSelectTransactionTest(sandBoxReadDB, fourthData); err != nil {
+			t.Fatalf("The transaction command failed: %v", err)
+		}
+	})
 }
```
Comment (julien-leclercq) [inline platform/database/tests/database_test.go:949]: wouldn't this be better in `sandbox_test.go`  ?

Comment (Kemosabert) [reply platform/database/tests/database_test.go:949]: you are right

**Review by ZFLloyd — APPROVED** [review]
Looks good 👍

**Pull request merged**